Tasks/Rusak/Task1/Task1-Deposit.py

```diff
@@ -0,0 +1,13 @@
+from decimal import Decimal
+
+deposit = Decimal('20000')
+period = 5
+rate = Decimal('15')/Decimal('100')
+
+rate_in_month = rate/Decimal('12')
+period_in_month = period*12
+
+for month in range(period_in_month):
+     deposit = deposit + (deposit*rate_in_month)
```

Comment:
an overkill for the problem, there is a specific formula to calculate such percentage